doc/releases/changelog-dev.md

```diff
@@ -242,62 +242,73 @@
   for theoretical background information on the general
   parameter-shift rule.
 
-* Continued development of the circuit-cutting compiler:
-
-  - A method for converting a quantum tape to a directed multigraph that is amenable
-    to graph partitioning algorithms for circuit cutting has been added.
-    [(#2107)](https://github.com/PennyLaneAI/pennylane/pull/2107)
-
-  - A method to replace `WireCut` nodes in a directed multigraph with `MeasureNode`
-    and `PrepareNode` placeholders has been added.
-    [(#2124)](https://github.com/PennyLaneAI/pennylane/pull/2124)
-
-  - A method has been added that takes a directed multigraph with `MeasureNode` and
-    `PrepareNode` placeholders and fragments into subgraphs and a communication graph.
-    [(#2153)](https://github.com/PennyLaneAI/pennylane/pull/2153)
-
-  - A method has been added that takes a directed multigraph with `MeasureNode`
-    and `PrepareNode` placeholder nodes and converts it into a tape.
-    [(#2165)](https://github.com/PennyLaneAI/pennylane/pull/2165)
-
-  - A differentiable tensor contraction function `contract_tensors` has been
-    added.
-    [(#2158)](https://github.com/PennyLaneAI/pennylane/pull/2158)
-
-  - A method has been added that expands a quantum tape over `MeasureNode` and `PrepareNode`
-    configurations.
-    [(#2169)](https://github.com/PennyLaneAI/pennylane/pull/2169)
-
-  - The postprocessing function for the `cut_circuit` transform has been added.
-    [(#2192)](https://github.com/PennyLaneAI/pennylane/pull/2192)
-    
-  - The `cut_circuit` transform has been added.
-    [(#2216)](https://github.com/PennyLaneAI/pennylane/pull/2216)
-
-  - A class `CutStrategy` which acts as an interface and coordinates device/user
-    constraints with circuit execution requirements to come up with the best sets
-    of graph partitioning parameters.
-    [(#2168)](https://github.com/PennyLaneAI/pennylane/pull/2168)
-
-  - A suite of integration tests has been added.
-    [(#2231)](https://github.com/PennyLaneAI/pennylane/pull/2231)
-    [(#2234)](https://github.com/PennyLaneAI/pennylane/pull/2234)
-    [(#2244)](https://github.com/PennyLaneAI/pennylane/pull/2244)
-    [(#2251)](https://github.com/PennyLaneAI/pennylane/pull/2251)
-    [(#2265)](https://github.com/PennyLaneAI/pennylane/pull/2265)
-
-  - Circuit fragments that are disconnected from the terminal measurements are now removed.
-    [(#2254)](https://github.com/PennyLaneAI/pennylane/pull/2254)
+* Support for circuit cutting has been added ✂️
+
+  Users can now run `N`-qubit circuits on devices with fewer than `N` qubits by strategically
```

Comment:
Can any circuit be cut?

Reply:
You can in principle cut any qubit circuit if you put the `WireCut` operations in the right places and are happy to wait long enough, but in practice certain circuits/circuit topologies will be more suited to efficiently circuit cutting.

Reply:
Interesting! Thanks :)

Reply:
Is it worth mentioning the restriction to `expval` measurements?